tools/rtrclient.c

```diff
@@ -230,6 +247,45 @@ static bool is_readable_file(const char *str)
 	return is_file(str);
 }
 
+#ifdef RTRLIB_HAVE_LIBSSH
+static bool is_utf8(const char *str)
+{
+	size_t len = strlen(str);
+
+	for (size_t i = 0; i <= len; ++i) {
```

Comment:
shouldn't this be `i < len` instead of `i <= len`?

Reply:
Yes. Though in practice i does not matter because the last byte is always the terminating null byte.
I fixed it anyway of course.

Reply:
Yes. fixed.